agcaballero/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,14 +41,22 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Retrieve max number of comments (if no info provided, default is 50)
+    int maxComments = Integer.parseInt(getParameter(request, "max-comments", "50"));
+
     // Retrieve from Datastore entities of type "Comment"
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     PreparedQuery results = datastore.prepare(query);
     StringBuilder commentDivs = new StringBuilder();
 
+    int count = 0;
+
     // Build a String of divs to hold comments to add to page
     for (Entity entity : results.asIterable()) {
+      // Check if max number of comments have been loaded
+      if(count == maxComments) break;
```

Comment:
Nit: newline before the break.

Reply:
I ended up removing this line to deploy a different fix. However, it's good to know that's stylistically better. Thank you!

Reply:
Done